README.md

```diff
@@ -1,73 +1,64 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Awesome Books
 
+This project is to build a basic website that allows users to add/remove books from a list.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
+- Visual Studio Code
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link](https://natanima.github.io/Awesome-books-Es6/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+$ git clone https://github.com/NatanimA/Awesome-books-Es6/
 
-### Setup
+$ git checkout -b feature-es6-refactoring
 
-### Install
-
-### Usage
+$ git pull origin feature-es6-refactoring
```

Comment:
- [OPTIONAL] Unfortunately the live demo does not work because it does not load the npm packages. Don't worry about it because you will learn how to deploy an application using Heroku or Netlify in the future. It would be nice if you mention in the 'Getting Started, 'Install" and 'Usage' sections that the user needs to clone the repo, run the `npm i` command and then load the app using something like the 'Live view' plugin.